### PR TITLE
scripts: Move node info into common sourceable scripts

### PR DESCRIPTION
**Pull request by chancez**

### CODE REVIEW

File: scripts/common.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+NODE1_NAME=node1
+NODE1_UUID=16e7d8a7-bfa9-428b-9117-363341bb330b
```
Comment (dghubble) [inline scripts/common.sh]: We don't require fixed UUID's for examples anymore. I'd remove that bit.

**Comment by dghubble** [comment]
Just the one cleanup, then looks good.

File: scripts/common.sh (hunk 1 above)
Comment (chancez) [inline scripts/common.sh]: Do they cause any problems? I think it could still be useful to have consistent UUIDs, even if they're not used in examples.

Comment (dghubble) [inline scripts/common.sh]: No, but if not required I don't want to pin them. Same reason we don't pin a serial id and other unused things.

**Comment by chancez** [comment]
@dghubble Updated

**Pull request merged**